src/components/Faq/FaqNav.js

```diff
@@ -4,7 +4,18 @@ import PropTypes from "prop-types";
 import "./FaqNav.css";
 import Button from "../Button";
 
-const FaqNav = ({ navData, pathname }) => {
+const FaqNav = ({ navData, pathname, lang }) => {
+  let pathRoot = `/refugee-guide`;
+  switch (lang) {
+    case "lt":
+      pathRoot = `/informacija-ukrainieciams`;
+      break;
+    case "ua":
+      pathRoot = `/refugee-guide`;
+      break;
+    //...
```

Comment:
Shouldn't we also add a `default` case, in case a `lang` prop is not supplied?

```
switch (lang) {
    case "lt":
      pathRoot = `/informacija-ukrainieciams`;
      break;
    case "ua":
      pathRoot = `/refugee-guide`;
      break;
    default:
      break;
```